packages/patternfly-4/react-core/src/components/Checkbox/Checkbox.md

```diff
@@ -46,6 +47,14 @@ class ControlledCheckbox extends React.Component {
           id="check-2"
           name="check2"
         />
+        <Checkbox
```

Comment:
would be good to wrap 2 checkboxes inside the checkbox and show the indeterminate state live. And can also get rid of several other duplicate examples

Reply:
Yeah! That's a good point. I like it!